<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1711</width>
    <height>1320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/main.png</normaloff>resources/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,0">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,4,2">
      <item row="1" column="0">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelHeader">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>resources/header.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QDockWidget" name="dockWidgetLog">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="floating">
       <bool>true</bool>
      </property>
      <property name="features">
       <set>QDockWidget::AllDockWidgetFeatures</set>
      </property>
      <property name="windowTitle">
       <string>日志</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEditLog"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>63</width>
     <height>63</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSetup"/>
   <addaction name="actionDisplayLog"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionNext"/>
   <addaction name="actionPrev"/>
   <addaction name="actionRedo"/>
   <addaction name="actionImageAdjust"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionSetup">
   <property name="icon">
    <iconset>
     <normaloff>resources/setup.png</normaloff>resources/setup.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;参数设置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionDisplayLog">
   <property name="icon">
    <iconset>
     <normaloff>resources/log.png</normaloff>resources/log.png</iconset>
   </property>
   <property name="text">
    <string>日志</string>
   </property>
   <property name="toolTip">
    <string>显示日志窗口</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>resources/exit.png</normaloff>resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionImageAdjust">
   <property name="icon">
    <iconset>
     <normaloff>resources/imgAdjust.png</normaloff>resources/imgAdjust.png</iconset>
   </property>
   <property name="text">
    <string>图像调整</string>
   </property>
   <property name="toolTip">
    <string>调整图像明暗度及对比度调整</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset>
     <normaloff>resources/start.png</normaloff>resources/start.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
   <property name="toolTip">
    <string>开始检测</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>resources/stop.png</normaloff>resources/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止检测</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset>
     <normaloff>resources/pause.png</normaloff>resources/pause.png</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="toolTip">
    <string>暂停检测</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset>
     <normaloff>resources/next.png</normaloff>resources/next.png</iconset>
   </property>
   <property name="text">
    <string>下一张</string>
   </property>
   <property name="toolTip">
    <string>检测下一张图像</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="icon">
    <iconset>
     <normaloff>resources/prev.png</normaloff>resources/prev.png</iconset>
   </property>
   <property name="text">
    <string>上一张</string>
   </property>
   <property name="toolTip">
    <string>检测上一张图像</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>resources/redo.png</normaloff>resources/redo.png</iconset>
   </property>
   <property name="text">
    <string>重新检测</string>
   </property>
   <property name="toolTip">
    <string>重新检测当前图像</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
